core/src/main/java/org/hisp/dhis/android/core/event/EventQueryCriteria.java

```diff
@@ -48,26 +45,21 @@
 import java.util.List;
 
 @AutoValue
-@JsonDeserialize(builder = $$AutoValue_EventQueryCriteria.Builder.class)
 public abstract class EventQueryCriteria extends FilterQueryCriteria implements CoreObject {
 
     @Nullable
-    @JsonProperty()
     @ColumnAdapter(IgnoreEventDataFilterListColumnAdapter.class)
     public abstract List<EventDataFilter> dataFilters();
```

Comment:
The class EventDataFilter keeps the annotations `@JsonProperty()`. They could be removed (unless this class is used somewhere else)